<launch>
  <arg name="num" default="1"/>
  <node pkg="laser_filters" type="scan_to_scan_filter_chain" name="$(eval 'laser_filter_' + str(arg('num')))">
      <rosparam command="load" file="$(find laser_filters)/turtlebot3_range_filter.yaml" />
      <remap from="/scan_filtered" to="$(eval '/robot' + str(arg('num')) + '/laserscan_filtered')" />
      <remap from="/scan" to="$(eval '/robot' + str(arg('num')) + '/scan')" />
  </node>

</launch>
